tests/logs/RegressionTests_hercules.log

```diff
@@ -31,400 +31,397 @@ Submodule hashes used in testing:
+PASS -- TEST cpld_control_p8.v2.sfc_intel [15:51, 15:04] (2213708 MB)
+PASS -- TEST cpld_restart_p8_intel [08:29, 07:34] (1995532 MB)
+PASS -- TEST cpld_control_qr_p8_intel [15:18, 14:27] (2228404 MB)
+PASS -- TEST cpld_restart_qr_p8_intel [08:49, 07:46] (1737308 MB)
+PASS -- TEST cpld_2threads_p8_intel [17:15, 16:28] (2563056 MB)
+PASS -- TEST cpld_decomp_p8_intel [15:12, 14:27] (2182832 MB)
+PASS -- TEST cpld_mpi_p8_intel [13:22, 12:48] (2101424 MB)
+PASS -- TEST cpld_control_ciceC_p8_intel [15:39, 14:51] (2208660 MB)
+PASS -- TEST cpld_control_c192_p8_intel [18:27, 17:03] (2992468 MB)
+PASS -- TEST cpld_restart_c192_p8_intel [07:58, 06:35] (2921260 MB)
+PASS -- TEST cpld_bmark_p8_intel [23:30, 18:35] (3852480 MB)
+PASS -- TEST cpld_restart_bmark_p8_intel [14:52, 09:17] (3644288 MB)
+PASS -- TEST cpld_s2sa_p8_intel [07:54, 07:08] (2100508 MB)
+
+PASS -- COMPILE s2sw_intel [11:38, 11:38](1 warnings,11 remarks)
+PASS -- TEST cpld_control_noaero_p8_intel [14:19, 13:34] (2022224 MB)
+PASS -- TEST cpld_control_nowave_noaero_p8_intel [10:59, 10:21] (2099716 MB)
+
+PASS -- COMPILE s2swa_debug_intel [06:42, 06:42](1410 warnings,1218 remarks)
+PASS -- TEST cpld_debug_p8_intel [22:12, 21:26] (2229196 MB)
+
+PASS -- COMPILE s2sw_debug_intel [04:42, 04:42](1410 warnings,1218 remarks)
+PASS -- TEST cpld_debug_noaero_p8_intel [09:03, 08:21] (2050340 MB)
+
+PASS -- COMPILE s2s_aoflux_intel [09:29, 09:29],3 remarks)
+PASS -- TEST cpld_control_noaero_p8_agrid_intel [05:21, 04:39] (2070624 MB)
+
+PASS -- COMPILE s2s_intel [09:42, 09:42](1 warnings,3 remarks)
+PASS -- TEST cpld_control_c48_intel [06:20, 05:53] (3038056 MB)
+PASS -- TEST cpld_warmstart_c48_intel [02:20, 01:50] (3012352 MB)
+PASS -- TEST cpld_restart_c48_intel [01:24, 01:04] (2474392 MB)
+
+PASS -- COMPILE s2swa_faster_intel [12:09, 12:09](1 warnings,11 remarks)
+PASS -- TEST cpld_control_p8_faster_intel [14:29, 13:54] (2192360 MB)
+
+PASS -- COMPILE s2sw_pdlib_intel [15:10, 15:10](1 warnings,11 remarks)
+PASS -- TEST cpld_control_pdlib_p8_intel [23:23, 22:44] (2070768 MB)
+PASS -- TEST cpld_restart_pdlib_p8_intel [12:04, 11:22] (1421444 MB)
+PASS -- TEST cpld_mpi_pdlib_p8_intel [26:18, 25:41] (1994064 MB)
+
+PASS -- COMPILE s2sw_pdlib_debug_intel [05:14, 05:14](1520 warnings,1988 remarks)
+PASS -- TEST cpld_debug_pdlib_p8_intel [28:21, 27:41] (2019272 MB)
+
+PASS -- COMPILE atm_dyn32_intel [09:39, 09:38](1 warnings,1 remarks)
+PASS -- TEST control_flake_intel [03:02, 02:51] (712540 MB)
+PASS -- TEST control_CubedSphereGrid_intel [03:04, 02:41] (1599404 MB)
+PASS -- TEST control_CubedSphereGrid_parallel_intel [04:08, 03:40] (1592960 MB)
+PASS -- TEST control_latlon_intel [03:05, 02:45] (1582116 MB)
+PASS -- TEST control_wrtGauss_netcdf_parallel_intel [03:12, 02:44] (1592800 MB)
+PASS -- TEST control_c48_intel [10:53, 10:34] (1704104 MB)
+PASS -- TEST control_c48.v2.sfc_intel [06:13, 06:01] (833868 MB)
+PASS -- TEST control_c192_intel [10:41, 10:11] (1754152 MB)
```

Comment:
What happened to the lnd_iau_intel test?

Reply:
I don't know...@jkbk2004 ?

Reply:
pushed hercules log again: test script sometimes not going thru fully between rocoto and log created on hercules.